### PR TITLE
Updates to config and other site details

### PR DESCRIPTION
**Pull request by dcwalk**
Aiming to address:

- [x] #10 (who @b5 AGPL, amirite?)
- ~#11 documentation RE: deployment~ (sitting on this for now!)
- [x] #27 remove extraneous libs, dependencies 
   - [x] footer scripts

### CODE REVIEW

**Comment by dcwalk** [comment]
@b5 -- Can you confirm AGPL was the intended license for the datatogether theme?

**Comment by dcwalk** [comment]
Hey @b5 -- can you let me know about the license? The documentation on the theme is a little ambiguous! Would be good to clear up :)

**Comment by dcwalk** [comment]
I'm gonna cut out the doc deployment as it looks like that might simplify, yaaaay! (#30 )

**Comment by dcwalk** [comment]
ping @b5 can you review that this is okay?

**Comment by b5** [comment]
Oh my gosh how did I miss this PR for so long.

I hate to be a stick in the mud, but can we go with GPL 3.0 if we're going to GPL? I have a really hard time getting around to liking AGPL.

File: config.toml [hunk 1]
```diff
@@ -1,35 +1,22 @@
-baseURL = "http://datatogether.org"
-relativeURLs = true
-languageCode = "en-us"
+baseURL = "https://datatogether.org"
```
Comment (b5) [inline config.toml:1]: This https switch _might_ cause some issues on builds, but I think it's worth a shot. Let's keep this change & if things explode we can revert

File: config.toml [hunk 2]
```diff
@@ -1,35 +1,22 @@
...
-# FaviconFile = "img/leaf.ico"
-# GATracker = "XYZ"
-# SegmentTracker= "b4iAxJT8ISitRFQ6qZGS9w7RTnaOpvju"
-# GitHub = "//github.com/qri-io"
-# HighlightJS = true
-# Initials = "ad" # Displayed on single post page; DEPRECATED in v0.3.0.
-# Lang = "en"
-# LinkedIn = "//linkedin.com/in/you"
-# Twitter = "//twitter.com/qri_io"
-# ExtraCssFiles = [ "/css/override.css" ] # In your `static/css`, add/remove files as necessary.
```
Comment (b5) [inline config.toml:23]: ty for removing this cruft :)

File: themes/datatogether/theme.toml [hunk 3]
```diff
@@ -2,20 +2,14 @@
 # See https://github.com/gohugoio/hugoThemes#themetoml for an example
 
 name = "Datatogether"
-license = "MIT"
-licenselink = "https://github.com/yourname/yourtheme/blob/master/LICENSE.md"
-description = ""
-homepage = "http://example.com/"
-tags = []
+license = "AGPLv3"
```
Comment (b5) [inline themes/datatogether/theme.toml]: I'd like to see this as GPL 3.0

**Review by b5 — DISMISSED** [review]
A few things here & there, otherwise this is a great set of changes, would love to see this merged.

**Review by b5 — APPROVED** [review]
WHAMMY

**Comment by dcwalk** [comment]
Okay, merging! Thanks for the review @b5

**Pull request merged**